src/platform/monitoring/external-services/config.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * List of external backend services that we monitor as defined by the VA API:
+ * https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml#L338
+ */
+export const EXTERNAL_SERVICES = {
+  appeals: 'appeals',
+  arcgis: 'arcgis',
+  dslogon: 'dslogon',
+  emis: 'emis',
+  // Enrollment System (HCA submissions)
+  es: 'es',
+  evss: 'evss',
+  // Intake, conversion, and mail handling services (central mail)
+  icmhs: 'icmhs',
+  // ID.me, identity provider
+  idme: 'idme',
+  // My HealtheVet
+  mhv: 'mhv',
+  // Master Veteran Index (source of veteran profile info)
+  mvi: 'mvi',
+  // Search.gov API
+  search: 'search',
+  // The Image Management System (education forms)
+  tims: 'tims',
+  // Vet360 - data source for centralized veteran contact information
+  vet360: 'vet360',
+  // Veteran ID Card v1
+  vic: 'vic',
+};
```

Comment:
Yes, I agree. I copied for now, and thought of just having it in one place once I found out more about how `backend_statuses` and `maintenance_windows` will be used going forward.

Reply:
@annaswims Will `backend_statuses` and `maintenance_windows` likely always share the same possible service values, as they do currently?

Reply:
[Based on the new settings config](https://github.com/department-of-veterans-affairs/vets-api/pull/3155/files#diff-646ad6b10917e4385cbcc8c8524e24a2R338) for the `backend_statuses` to work, I believe they're all the same as the `maintenance_windows` services.

Reply:
yes, though there are some services in backend_statuses that are not listed in maintenance_windows.  I think we should probably fix that.

Reply:
It would be really cool to see the code reflect the services in backend_statuses as a superset of those in maintenance_windows. We have service constants in a few places, and would love to see their relationships defined.